examples/version-migration/separate-container/src/start.ts

```diff
@@ -77,84 +132,51 @@ const renderModel = (model: IVersionedModel, migrationTool: IMigrationTool): voi
 	}
 };
 
-async function start(): Promise<void> {
-	const loader = new SimpleLoader({
-		urlResolver: new InsecureTinyliciousUrlResolver(),
-		documentServiceFactory: new RouterliciousDocumentServiceFactory(
-			new InsecureTinyliciousTokenProvider(),
-		),
-		codeLoader: new DemoCodeLoader(),
-		generateCreateNewRequest: createTinyliciousCreateNewRequest,
+export const setupContainer = async (
+	id: string,
+	alreadyLoadedContainer?: IContainer | undefined,
+): Promise<void> => {
+	// The first createDetached flow ends up with a live container reference that we want to retain rather
+	// than disposing it and loading a second time.  In all other cases we'll do the actual load here.
+	const container =
+		alreadyLoadedContainer ??
+		(await loadExistingContainer({ ...loaderProps, request: { url: id } }));
+	const model = await getModelFromContainer<IMigratableModel>(container);
+
+	// In this example, our container code mixes in an IMigratorEntryPoint to the container entryPoint.  The getMigrator
+	// function lets us construct an IMigrator by providing the necessary external tools it needs to operate.  The IMigrator
+	// is an object we can use to watch migration status, propose a migration, and discover the migration result.
+	const { getMigrator } = (await container.getEntryPoint()) as IMigratorEntryPoint;
+	const migrator: IMigrator = await getMigrator(
+		async () => loadExistingContainer({ ...loaderProps, request: { url: id } }),
```

Comment:
`loadExistingContainer({ ...loaderProps, request: { url: id } }` is popular.
This is second in here.
So why would returning `container` be bad for the lambda? Called a second time (in load case) gets a second container and that is needed?

Reply:
Yes, the `LoadSourceContainerCallback` needs to be a separate copy of the container because `container` may have local un-ack'd changes (similar to how the summarizer client runs on a separate container instance).

Reply:
Suggest commenting that here in the example.

Reply:
Added comment in latest